### ISSUE TITLE
English help

### ISSUE THREAD

**Issue by Serg-Norseman** [Serg-Norseman/GEDKeeper#4]
Translate help in English.

**Comment by ruslangaripov** [comment]
I'm definitely in! :smiling_imp:

**Comment by ruslangaripov** [comment]
WOW! :laughing: GitHub can see references to a ticket made from a fork repository! I unbonnet! :tophat:

**Comment by Serg-Norseman** [comment]
Your help is very valuable for me! Thank you! :)

**Comment by ruslangaripov** [comment]
Be a part of this project is a honor for me! :smiling_imp:

**Comment by Serg-Norseman** [comment]
As soon as you can - add yourself to "authors"

**Comment by ruslangaripov** [comment]
> As soon as you can - add yourself to "authors"
Done for English help content.

**Comment by Serg-Norseman** [comment]
Good :) In the page in Russian I'll add you :)

**Comment by ruslangaripov** [comment]
Oh My God! I've killed the ticket's log :laughing: :sob: :joy: :feelsgood: :gun:

I rebased the branch (`users/brigadir/ticket4`) twice, but I have just noticed **THIS FLOW**!

> WOW! :laughing: GitHub can see references to a ticket made from a fork repository! I unbonnet! :tophat:

No! Stop it! I don't want this anymore!

Sorry :hankey: I really haven't thought this might happen. I should stop refer to this ticket in the related commits. If I'll stop rebasing, we will end up with a rush of merge commits. I have made :hankey: here but I don't want to make it in the log.

**Comment by ruslangaripov** [comment]
I have decided to finish with bloating in this topic :neutral_face:

1. No "general" commit refers this ticket anymore. Only a specific one.
2. I have completely **rewritten** and **rebased** 'users/brigadir/ticket4' topic branch. Because of the first item (no commit refers this issue) future rebasing generally will not add reference messages here.
3. I have decided to replace my work email address and user name to change how I identify myself to local git tool-chain at work. I do not want to reveal work identities anymore. But not because this is prohibited by my employer, no. Just because I do not want my employer to see my contributions here. At least by an obvious way. The previous work identity (*GaripovRN@ukbtm.ru*) remains in repositories and in my GitHub account. But starting from today I switch to '*garipov.rn@yandex.ru*' email address and '*#Ruslan Garipov*' user name. The '#' stands for *jail*. This is what my current workplace is.
See [3a16696 patch](https://github.com/ruslangaripov/GEDKeeper/commit/3a1669607ea92037d8fc5c25882147d3d4b50108.patch) for example of my new identities.
4. All commits were signed by me.

Once again, I am sorry for :hankey: I have made here :finnadie:

You live and learn! :feelsgood:
